lib/src/git.rs

```diff
@@ -1259,7 +1259,7 @@ pub fn reset_head(mut_repo: &mut MutableRepo, wc_commit: &Commit) -> Result<(),
 
     // If the first parent of the working copy has changed, reset the Git HEAD.
     let old_head_target = mut_repo.git_head();
-    if old_head_target != new_head_target {
+    if old_head_target != new_head_target || true {
```

Comment:
Did you intend to leave the `|| true` there? If so, what is accomplishing?

Reply:
Oh, that was just for testing. I'll remove it